src/ffi/message.rs

```diff
@@ -23,11 +28,11 @@ pub enum Event {
         user: String,
     },
 }
-#[uniffi::export]
+//#[uniffi::export]
 impl Event {
     /// Creates an [Event] from a string and ensures that special Phoenix control events are turned
     /// into [Event::Phoenix] while others are [Event::User].
-    #[uniffi::constructor]
+    //#[uniffi::constructor]
```

Comment:
Like `JSON` this is code that doesn't include my current understanding that only `uniffi::Object` can have `uniffi::constructor`. Since `Event` is a `uniffi::Enum`, this `uniffi::constructor` should be removed.